pkgs/development/tools/misc/openfpgaloader/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+
+  libftdi1,
+  libudev,
+  libusb,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openfpgaloader";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "trabucayre";
+    repo = "openFPGALoader";
+    rev = "v${version}";
+    sha256 = "0j87mlghbanh6c7lrxv0x3p6zgd0wrkcs9b8jf6ifh7b3ivcfg82";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [
+    libftdi1
+    udev
```

Comment:
```suggestion
    libudev
```

Reply:
This causes ofborg failures: `libudev` is an alias of `udev`, and apparently ofborg wants me to use the non-alias. I originally sent this PR with `libudev`, but CI failed and chided me for using an alias, so I switched to `udev` *shrug*

Reply:
You have libudev in inputs and use udev here. I think you forgot to update your inputs.

Reply:
Oops, fixed.